### PR TITLE
Add Input Mask

### PR DESCRIPTION
**Pull request by jadsondorneles**
If applied, this pull request will add Input Mask

### Card Link:
N/A

### Design Expected Screenshot
N/A

### Implementation Screenshot or GIF
agnostic

### CODE REVIEW

File: src/components/Input/index.tsx [hunk 1]
```diff
@@ -176,39 +179,70 @@ const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
     return '';
   };
 
+  const displayInput = () => {
+    const attributeValueGatherer = value => {
```
Comment (kaiomagalhaes) [inline src/components/Input/index.tsx:183]: please rename to getAttr

File: package.json [hunk 2]
```diff
@@ -96,10 +96,12 @@
     "@tiptap/extension-table-row": "^2.0.0-beta.14",
```
Comment (kaiomagalhaes) [inline package.json:96]: please squash your commits

**Review by kaiomagalhaes — CHANGES_REQUESTED** [review]

File: src/components/Input/index.tsx [hunk 3]
```diff
@@ -48,6 +49,7 @@ export interface Props {
   variablesClassName?: string;
   withActionIcon?: boolean;
   withPrependSeparator?: boolean;
+  mask?: string;
```
Comment (paulohfev) [inline src/components/Input/index.tsx:52]: @viniciusaa please order the types alphabetically

File: src/components/Input/index.tsx [hunk 4]
```diff
@@ -84,6 +86,7 @@ const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
     validations,
     withPrependSeparator,
     withActionIcon,
+    mask,
```
Comment (paulohfev) [inline src/components/Input/index.tsx:89]: please order the properties alphabetically

File: src/components/Input/index.tsx (hunk 1 above)
Comment (paulohfev) [reply src/components/Input/index.tsx:183]: @viniciusaa please address this and rename the function as mentioned.

File: src/components/Input/index.tsx [hunk 5]
```diff
@@ -176,39 +179,70 @@ const Input = React.forwardRef<HTMLInputElement, Props>((props, ref) => {
...
+        })}
+        onBlur={attributeValueGatherer(handleBlur)}
+      />
+    );
+  };
+
+  const displayMaskedInput = mask => {
+    return (
+      <ReactInputMask
+        mask={mask}
```
Comment (paulohfev) [inline src/components/Input/index.tsx:222]: @viniciusaa please order the properties alphabetically.

File: src/components/Input/Input.stories.tsx [hunk 6]
```diff
@@ -26,6 +26,13 @@ Default.args = {
   label: 'Input Label'
 };
 
+export const WithMask = Template.bind({});
+WithMask.args = {
+  placeholder: 'MM / DD / YYYY',
+  label: 'Input Date for Mask',
+  mask: '99/99/9999'
```
Comment (paulohfev) [inline src/components/Input/Input.stories.tsx:33]: @viniciusaa should this value be displayed in the input upon user interaction?

**Review by paulohfev — CHANGES_REQUESTED** [review]

File: src/components/Input/Input.stories.tsx (hunk 6 above)
Comment (viniciusaa) [reply src/components/Input/Input.stories.tsx:33]: No, this value determines that the input will accept only numbers and the "/" will be permanent and non-editable (check the documentation that I have added).

File: src/components/Input/index.tsx (hunk 4 above)
Comment (paulohfev) [reply src/components/Input/index.tsx:89]: @viniciusaa all that's missing is this adjustment.

**Review by paulohfev — CHANGES_REQUESTED** [review]

**Pull request merged**